server/chat-plugins/randombattles/index.ts

```diff
@@ -173,6 +174,21 @@ function getData(species: string | Species, format: string | Format): any | null
 	return data;
 }
 
+/**
+ * Gets the random battles data for a Pokemon for doubles where doubles is its own file
+ * but it still uses the old system. Currently only gen7dubs.
+ */
+function getStandaloneDoublesData(species: string | Species, format: string | Format): any | null {
```

Comment:
Name's a bit confusing - maybe roll this functionality into one function for all of doubles?

Reply:
Since gen 7 randdubs will continue using the old system, I believe that folding it into `getData()` would be the simplest way to go about it.